scripts/new-meetup.js

```diff
@@ -33,29 +30,29 @@ module.exports = function(robot) {
       }
 
       try {
-        result = JSON.parse(body).results;
+        const result = JSON.parse(body).results;
+        if(result && result.length > 0) {
+          const lastResult = robot.brain.get('lastResult') || [];
+          result.forEach(function(meetup){
+            const key = toKey(meetup);
+            if (!lastResult.some(prev => prev === key)) {
```

Comment:
What happens if the last message was the exact same list, but it happened 7 days ago? Currently it would just stay silent, which might lead to confusion, as 7 days later is a valid time to re-broadcast the same message.

The 7 days was an arbitrary choice by me, not sure what the ideal cutoff is.

Reply:
Not sure what you mean? This should only announce when there is a brand new meetup available.